service/controller/chartconfig/v7/resource/chart/create.go

```diff
@@ -6,16 +6,27 @@ import (
 	"fmt"
 
 	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/controller/context/resourcecanceledcontext"
 	"k8s.io/helm/pkg/helm"
 
-	"github.com/giantswarm/chart-operator/service/controller/chartconfig/v6/key"
+	"github.com/giantswarm/chart-operator/service/controller/chartconfig/v7/key"
 )
 
 func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
 	customObject, err := key.ToCustomObject(obj)
 	if err != nil {
 		return microerror.Mask(err)
 	}
+
+	if key.IsCordoned(customObject) {
```

Comment:
With the CRUD resource implementations we only need to cancel in `GetCurrentState` as this is the entrypoint for all informer events being dispatched.

Reply:
Good to know, I'll remove this part.